tests/cxx/inheriting_ctor-i1.h

```diff
@@ -0,0 +1 @@
+#include "inheriting_ctor-i2.h"

```

Comment:
This needs a license header and include guard

Reply:
Also, I think this should be called `...-d1.h` -- I believe 'd' is for 'direct' and 'i' is is for 'indirect'.